OracleDatabase/18.4.0-XE/scripts/install.sh

```diff
@@ -19,6 +19,9 @@ echo 'INSTALLER: Started up'
 # get up to date
 yum upgrade -y
 
+# ensure wget is installed for downloading database installer
+yum install -y wget
```

Comment:
Instead of installing `wget`, perhaps consider using `curl` which is installed by default?

Reply:
@Djelibeybi `wget` is already installed, too. I just didn't want the script to break if that changed in a future version of the base box.

Reply:
Ha, fair enough. Goes to show I should probably do a _modicum_ of verification before reviewing things. :) Good job.

Reply:
While I used `wget` in the past, I now prefer using `curl` -- `yum` (and many others) depends on `curl`, so you can rely on the fact it is already installed as @Djelibeybi says...

@PaulNeumann: don't change anything, this is just an FYI

Reply:
Right. I switched from `yum` to `curl` after developing the `oraclelinux:7-slim` image (that doesn't have `wget`).

Reply:
@Djelibeybi @AmedeeBulle Thank you. It sounds like using `curl` would be preferable, so I'd be happy to do that. The download code would then be (tested and ShellChecked):

```bash
if [[ ! -f /vagrant/"${db_installer}" ]]; then
  echo 'INSTALLER: Downloading Oracle Database software'
  (
    cd /vagrant || exit 1
    curl -L -O -s https://download.oracle.com/otn-pub/otn_software/db-express/"${db_installer}"
  )
fi
```

Reply:
Works for me!

Reply:
Done. Thanks again!